lib/ace/mouse/default_gutter_handler.js

```diff
@@ -87,6 +87,7 @@ function GutterHandler(mouseHandler) {
         tooltipAnnotation = annotation.text.join("<br/>");
 
         tooltip.setHtml(tooltipAnnotation);
+        tooltip.setClassName(annotation.className.trim());
```

Comment:
It might happen that className is undefined or null when annotation type is custom for example, so we should account for this too